demo-spring-testing/src/main/java/com/examples/spring/demo/DemoSpringTestingApplication.java

```diff
@@ -7,6 +7,11 @@
 public class DemoSpringTestingApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(DemoSpringTestingApplication.class, args);
+		try {
+			SpringApplication.run(DemoSpringTestingApplication.class, args);
+		} catch (IllegalArgumentException e1) {
+		} catch (Throwable e) {
+//			e.printStackTrace();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)